include/drivers/bluetooth/hci_driver.h

```diff
@@ -50,8 +50,10 @@ static inline bool bt_hci_evt_is_prio(u8_t evt)
 	switch (evt) {
 	case BT_HCI_EVT_CMD_COMPLETE:
 	case BT_HCI_EVT_CMD_STATUS:
+		/* fallthrough */
```

Comment:
> This seems like an odd addition. Why is it needed for CMD_STATUS but none of the other case statements?

Yes, I agree - ask the checker, it kept reporting error on missing "fallthrough"

Reply:
Probably it gets confused by the preprocessor macro, thinking there's a "body" to the case statement even though there isn't.

Reply:
It's the ifdef, I have seen it before. It is only a warning though, So I have usually ignored it.

Reply:
Yeah, I just saw the same thing when working with he remote info PR. I agree, we should just ignore it. Since I already merged this I wouldn't mind a follow-up PR to remove it 😉